src/stories/Button/component.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { action } from "@storybook/addon-actions";
+import StyledButton from "./index";
+
+const defaultText = "Default";
+
+export default {
+    title: "Button",
+    component: StyledButton,
+};
+
+export const Default = () => <StyledButton onClick={action("clicked")}>{defaultText}</StyledButton>;
```

Comment:
We don't need to export anything from our stories as this is used exclusively for storybook.
You would import the StyledButton component to use in your react app just like you've imported here.

Reply:
changed to storiesOf